.github/workflows/foundry-tests.yml

```diff
@@ -0,0 +1,47 @@
+name: Tests in foundry
+
+on:
+  workflow_call:
+    secrets:
+      HARDHAT_FORKING_URL:
+        required: true
+
+jobs:
+  foundry-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
Please change to commit instead of tag:
```       
# from tag: v3.0.0
uses: actions/checkout@a12a3943b4bdde767164f792f33f40b04645d846
```
Please add name also:
```
- name: Checkout
```